server/routes/users.js

```diff
@@ -0,0 +1,17 @@
+const userController = require('../controllers').users;
+
+module.exports = (app) => {
+  app.post('/users', userController.create);
+  app.get('/users', userController.list);
+  //app.post('/users/login', userController.login);
+  app.get('/search/users/', userController.searchUser);
+  app.get('/users/:userId', userController.retrieveOne);
+  app.get('/limit/users/', userController.retrieveLimited);
```

Comment:
**Code Quality - Nitpick**

- Retrieving limited users shouldn't be its own endpoint. Add the logic for it on the `/users?limit=10` with a query like so.